INN/umbrella-citylimits
# Create widget based on The Neighborhoods map

## Changes

This pull request makes the following changes:

- Creates an ugly widget titled `City Limits Neighborhoods Map` to display the The Neighborhoods map and data

![Screen Shot 2019-11-15 at 2 48 47 PM](https://user-images.githubusercontent.com/18353636/68971218-0c993c00-07b7-11ea-97d7-6bc7019ebdb8.png)

## Why



It was part of a template, now it can be used anywhere!

## Testing/Questions

Steps to test this PR:

1. Add this widget somewhere
2. Verify it works and looks moderately close to how it does on production
